client/js/lounge.js

```diff
@@ -227,6 +227,10 @@ $(function() {
 		var chan = chat.find(target);
 		var msg;
 
+		if (highlights.some((h) => data.msg.text.indexOf(h) > -1)) {
```

Comment:
Feels a bit unnecessary to include ES6 syntax - and effectively breaking 45 % of global browser market share - for something so small. Could be an opportunity to add babel transpilation though.